keanu-project/src/test/java/io/improbable/keanu/e2e/eight/EightSchools.java

```diff
@@ -0,0 +1,67 @@
+package io.improbable.keanu.e2e.eight;
+
+import io.improbable.keanu.algorithms.NetworkSamples;
+import io.improbable.keanu.algorithms.mcmc.NUTS;
+import io.improbable.keanu.network.BayesianNetwork;
+import io.improbable.keanu.tensor.dbl.DoubleTensor;
+import io.improbable.keanu.vertices.ConstantVertex;
+import io.improbable.keanu.vertices.dbl.DoubleVertex;
+import io.improbable.keanu.vertices.dbl.KeanuRandom;
+import io.improbable.keanu.vertices.dbl.probabilistic.GaussianVertex;
+import io.improbable.keanu.vertices.dbl.probabilistic.HalfCauchyVertex;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.Arrays;
+
+@Slf4j
+public class EightSchools {
```

Comment:
Can we add some docs explaining what this is

Reply:
I'll add a link to it. I first need to ask if we can use it because it's code that was posted on our github issues.